CONTRIBUTING.md

```diff
@@ -0,0 +1,107 @@
+* v1.26.9-rancher1-1
+* v1.25.14-rancher1-1
+* v1.24.17-rancher1-1
+* v1.23.16-rancher2-3
+
+whereas [release v1.5.0](https://github.com/rancher/rke/releases/tag/v1.5.0) includes:
+* v1.25.14-rancher1-1
+* v1.26.9-rancher1-1
+* v1.27.6-rancher1-1
+
+If a Kubernetes version is supported, that means we can deploy a cluster with that Kubernetes version using the binary included in that RKE release.
+
+When doing a commit in RKE code, we must carefully think if we want that commit in both tracks and hence backported. Sometimes, there is a reason to no backport it, for example a flag, that we would like to limit to only the newer track. However, that is exceptional and normally commits end up in both branches as a way to reduce the differences between those and simplify RKE maintainability. Therefore, usually we backport a commit between both RKE tracks. In that case, the following steps should be applied:
+
+* Create an Issue in rancher/rancher github project for Project Managers and QA to track the commit. Set the decided milestone for release/v1.x in the issue
+* Create another Issue in the same github project and in the description, specify it is a backport of the previous commit. Set the decided milestone for release/v1.(x-1) in the issue
+* Create the PR with the commit in branch release/v1.x and in the description, link the newly created issue
+* Cherry-pick the commit in branch release/v1.(x-1) and in the description, link the created issue for that RKE track.
+
+A commit is normally available in all supported Kubernetes versions of the release unless scoped in the code to a specific version. Here are two examples for this:
+* Available in all supported Kubernetes versions: https://github.com/rancher/rke/commit/1dad2582e1efe5564b6d1a7385afad70d5dbd1f7
+* Scoped to a specific version: https://github.com/rancher/rke/commit/feca6a3048c1f8c9031ab2849a46c46a7ea1d673
+
+Note that in the scoped case, there is a check on the version, which is normally fetch by executing the line `util.StrToSemVer(k8sVersion)`. In this case, as the same code is executed for all Kubernetes versions, unless scoped, there is no need to create extra PRs to effectively backport the code to all Kubernetes versions.
+
+
+#### Add-ons
+
+RKE consumes different add-ons to deploy a usable Kubernetes cluster, e.g. CNI plugins. Typically all Kubernetes versions include the same range of add-ons but use different add-on versions. For example, for the flannel add-on:
+* In v1.26.8-rancher1-1 flannel version is `v0.21.4`
+* In v1.24.17-rancher1-1 flannel version is `v0.15.1`
+
+What add-on version is used per RKE Kubernetes release is mapped in the `data.json` file, which is a file autogenerated by querying [KDM](https://github.com/rancher/kontainer-driver-metadata) releases. Read the RKE [README.md](https://github.com/rancher/rke/blob/release/v1.4/README.md) for more information.
```

Comment:
Suggesting to use relative link so we're not pointing to specific branch (default branch is already `release/v1.5` while this URL points to `release/v1.4`).

Reply:
What do you mean by relative link? I thought I always need to specify the release in the url to access the file (README.md in this case)

Reply:
Could you link this to templates.go file that maps the versions to k8s versions instead?